e2e/tests/init_test.go

```diff
@@ -76,20 +95,108 @@ func (s *suite) TheUserSendsUpoktToAnotherAddress(amount int64) {
 	s.pocketd.result = res
 }
 
-func (s *suite) getAddresses() [2]string {
-	var strs [2]string
+func (s *suite) TheAccountHasABalanceGreaterThanUpokt(acc string, amount int64) {
+	s.buildAddrMap()
```

Comment:
Rathe than doing this before the beginning of each step, thoughts on putting it into the `Before` function?